implement api client with auth bearer (#6)
Alex-Izquierdo/ansible-events-qa: docs/development.md

# Development guide

## Api tests

> The most of the cases, to write some test for the API you will need the api client to make requests

Reviewer: ```suggestion
In most cases, to write a test for the API you will need to import `api_client` in order to make requests:
```

```python
from ansible_events_qa.api import api_client
def test_something():
    response = api_client.current_user.show()
    assert response.status_code == 200
```

```python
from ansible_events_qa.api import api_client
def test_something(create_five_projects):
    response = api_client.projects.list()
    assert len(response.data) == 5
```

> By default, it uses the `default_user` defined in the settings. You can use the constructor to use the client with another user:

Reviewer: ```suggestion
By default, it uses the `default_user` defined in settings.yml. You can use the constructor to instantiate the client with a different user:
```
